### ISSUE TITLE
CMD + A doesn't work in Mac

### ISSUE THREAD

**Issue by hamzaydia** [defold/editor2-issues#1922]
Hi, CMD + A to select all is not working.

**Comment by erikangelin-king** [comment]
Do you mean when editing gameobjects, collections and guis? Seems to work fine in the code editor.

**Comment by hamzaydia** [comment]
No, I mean in the code editor, I have AZERTY keyboard when I click `⌘A` It's closes the program.

**Comment by sicher** [comment]
That is because the editor reads raw qwerty so you get a CMD + Q. It's not currently considering other keymaps. 

See #1236
